### PR TITLE
Feat/303 timer styling

### LINKED ISSUE #303: style(cooking-mode): improve cooking mode styling
### Suggestions from Dec 5 meeting

- [ ] Need margin for the buttons
- [ ] Wider buttons
- [ ] Top margin for all boxes
- [ ] Bigger Font Size
- [ ] Space after colons
- [ ] For Timer
    - [ ] make hours, minutes, seconds button and input bigger
        add spacing

### LINKED ISSUE #314: fix(cooking-mode): rename hands-free to cooking-mode
Rename everything in our code base that uses hands-free to the updated name, cooking-mode

### PR DESCRIPTION
**Pull request by phamavery**
Fixing timer styling. Some notes:
- I did not align the timer with the directions. This would require putting the timer and directions in the same div (basically have one giant div) and things were getting really messy so I decided to leave it
- This PR also contains small changes throughout the code base where I renamed stuff from hands-free to cooking-mode. This includes changes to the router and unit tests. I did not change any core logic (hopefully), just naming changes

Closes #303, closes #314

### CODE REVIEW

**Review by jbutera94 — APPROVED** [review]
Looks good to me. Not sure why the tests are failing, I'll look into that right now.

**Comment by phamavery** [comment]
> Looks good to me. Not sure why the tests are failing, I'll look into that right now.

Yeah, I have no idea why. All i did was change the name to cooking-mode and now it's null for some reason

**Comment by keskinmbaha** [comment]
Hey Avery, great job on the styling! However, I think a few things seemed to still be happening:

- The next and back buttons still touch each other when fullscreen on a 27-inch monitor on chrome.
Screen Shot 2021-12-06 at 4 31 10 PM
- The text field buttons are still very small (as the screenshot shows). Can you make them, as a possible suggestion, like the size of the actual buttons? Don't know how that would look though.
- Only allow numbers. The text fields right now are allowing other input like -, ., and e. the '-' sign is actually causing the timer to count down from 24 - x (x being the user input)
- Combining the two buttons into one, which I think we forgot to add this to the issue during the meeting.

**Comment by keskinmbaha** [comment]
Also, the e2e test is failing because when the page loads, all the screen can see is the video. It cannot see the cooking-mode button:

image

So once Justin moves the video down to the directions, then the cooking mode button should be visible and then the test should pass.

**Comment by jbutera94** [comment]
I'll get that fixed and merged in tonight so the test passes

**Comment by jbutera94** [comment]
@phamavery The video issue is fixed, so once you're ready with the new suggestions from slack, please merge the latest `develop` into your branch

**Comment by keskinmbaha** [comment]
Hey Avery, nice job, but sadly I got a few more bugs:

- Start and reset buttons are next to each other (half screen on 27inch monitor with chrome)
Screen Shot 2021-12-06 at 11 22 51 PM

- The hover animation over the start/pause and reset buttons is not smooth unlike the rest of the UI. Checkout out the contribute page for an example.

- Able to cause the timer to bug out if I put too high of a number in the text fields. Besides being blank I also received other weird input, like at one point the timer had the letter 'T' instead of a number. Another time it just copied the previous **correct** input I gave it:

https://user-images.githubusercontent.com/62824264/144987184-70be764f-3762-4d7e-a094-39aedb2dc5e3.mov

- Somehow, when starting and pausing on 0 seconds, I am able to loop back around to 24hrs even though the input timer was set at 1 second. Here is a video:

https://user-images.githubusercontent.com/62824264/144986345-c7731a1a-0138-4bce-aab2-51ea199f9f4d.mov

- As a suggestion, if you can without spending too much time, make the text field numbers font size bigger so its consistent with the size of the text field box.

Thanks again Avery.

**Comment by keskinmbaha** [comment]
One last comment, video and picture are slightly misaligned:
Screen Shot 2021-12-06 at 11 54 49 PM

**Comment by phamavery** [comment]
Is a split screen a resolution we should be concerned about though? I thought we were gonna the weird CSS issues at a certain point

**Comment by keskinmbaha** [comment]
I believe a half screen on the 27inch monitor is more than 1024px. Since its a 2560 x 1440p display.

File: source/components/cooking-mode/cooking-mode.js [hunk 1]
```diff
@@ -25,6 +25,8 @@ class HandsFree extends YummyRecipesComponent {
    */
   async setupElement() {
     this.shadowRoot.getElementById("timer-display").style.visibility = "hidden";
+
+    //console.log(this.shadowRoot.getElementById("pause-button").style.display);
```
Comment (jbutera94) [inline source/components/cooking-mode/cooking-mode.js:29]: Please remove this line if not needed.

**Review by jbutera94 — APPROVED** [review]
Looks good, left one super tiny comment.

**Review by jbutera94 — APPROVED** [review]
Looks good!

**Pull request merged**